src/aead/aes_gcm.rs

```diff
@@ -172,6 +172,9 @@ mod tests {
     }
 
     extern "C" {
+        // AES_set_encrypt_key returns 0 on success, unlike most BoringSSL functions.
+        // https://github.com/google/boringssl/blob/28babde159253bfa9003a445242605806fff5f1f/include/openssl/aes.h#L83
```

Comment:
Please remove the link to the BoringSSL code. If the comment being linked to is missing in *ring* then we should add the comment to *ring*.

Reply:
I added a comment to ring. I'm going to fix [this issue](https://github.com/briansmith/ring/issues/113) next, making it irrelevant.